nautobot_device_lifecycle_mgmt/metrics.py

```diff
@@ -0,0 +1,57 @@
+"""Nautobot Device LCM  plugin application level metrics ."""
+from datetime import datetime
+
+from django.conf import settings
+from nautobot.dcim.models import Device, Site, InventoryItem
+from prometheus_client.core import GaugeMetricFamily
+
+from nautobot_device_lifecycle_mgmt.models import HardwareLCM
+
+PLUGIN_SETTINGS = settings.PLUGINS_CONFIG.get("nautobot_device_lifecycle_mgmt", {})
+
+
+def nautobot_metrics_dlcm_eos():
+    """Calculate number of EOS devices per Device Type and per Site.
```

Comment:
```suggestion
    """Calculate number of End-of-Support (EOS) devices per Device Type and per Site.
```
I think it'll be clearer to those not familiar with the term (which is anyway a bit ambiguous as it could also mean End-of-Sales).